ch3/download_model_fluent.py

```diff
@@ -0,0 +1,47 @@
+import os
+import sys
+
+import mlflow
+import pandas as pd
+import psycopg2
+from sklearn.metrics import accuracy_score
+from sklearn.model_selection import train_test_split
+
+# 0. set mlflow environments
+MLFLOW_TRACKING_URI = "http://localhost:5000"
+MLFLOW_S3_ENDPOINT_URL = "http://localhost:9000"
+
+os.environ["MLFLOW_S3_ENDPOINT_URL"] = MLFLOW_S3_ENDPOINT_URL
+os.environ["MLFLOW_TRACKING_URI"] = MLFLOW_TRACKING_URI
+os.environ["AWS_ACCESS_KEY_ID"] = "mystorage"
+os.environ["AWS_SECRET_ACCESS_KEY"] = "mystoragepw"
+
+# 1. get data
+db_connect = psycopg2.connect(
+    host="localhost",
+    database="postgres",
+    user="postgres",
+    password="mypassword",
+)
+df = pd.read_sql("SELECT * FROM iris_data ORDER BY id DESC LIMIT 10", db_connect)
```

Comment:
이 부분은 왜 limit 10으로 들어가있는건가요?

Reply:
```suggestion
df = pd.read_sql("SELECT * FROM iris_data ORDER BY id DESC LIMIT 100", db_connect)
```
이거 100인데 제가 잘못 적어놨습니다 ㅎㅎ;

Reply:
100으로 바꾸겠습니다 :)